README.md

```diff
@@ -92,9 +92,15 @@ If the `aws_tag_select` feature was used, an additional information metric will
 # TYPE aws_resource_info gauge
 aws_resource_info{job="aws_elb",instance="",arn="arn:aws:elasticloadbalancing:eu-west-1:121212121212:loadbalancer/mylb",load_balancer_name="mylb",tag_Monitoring="enabled",tag_MyOtherKey="MyOtherValue",} 1.0
 ```
-
 All metrics are exported as gauges.
 
+aws_recource_info can be joined with other metrics using group_left in PromQL such as following:
```

Comment:
such as the following

Reply:
@brian-brazil Let's hope I got it right this time :)

Reply:
You didn't add the "the"